pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManagerTest.java

```diff
@@ -717,7 +743,7 @@ protected void start() {
     protected boolean consumeLoop()
         throws Exception {
       if (_throwExceptionFromConsume) {
-        throw new PermanentConsumerException(Errors.OFFSET_OUT_OF_RANGE.exception());
+        throw new PermanentConsumerException(new Throwable("Offset out of range"));
```

Comment:
why wrap exception within an exception?

Reply:
Not sure why the test was written that way in the first place. I'm just trying to remove dependency on the kafka class Errors.
PermanentConsumerException wraps stream specific exceptions

Reply:
To add to Neha's comment, the class being tested behaves differently based on whether PermanentConsumerException is thrown or transient one is thrown. It is up to the stream to decide if an exception is permanent or transient. The higher layer has no idea. So we need to wrap.

Reply:
Got it.